lxd/db/cluster/schema.go

```diff
@@ -155,79 +153,71 @@ CREATE TABLE "instances_profiles" (
     profile_id INTEGER NOT NULL,
     apply_order INTEGER NOT NULL default 0,
     UNIQUE (instance_id, profile_id),
-    FOREIGN KEY (instance_id) REFERENCES "instances"(id) ON DELETE CASCADE,
-    FOREIGN KEY (profile_id) REFERENCES profiles(id) ON DELETE CASCADE
-);
-CREATE INDEX instances_project_id_and_name_idx ON instances (project_id,
```

Comment:
Whats happened here? The indexes appear to have been removed?

Reply:
:grimacing: 

They get dropped when the tables are dropped. Thought I had recreated them all but clearly I have missed a few.

Reply:
Fixed